sentry_telegram/plugin.py

```diff
@@ -131,9 +145,19 @@ def get_receivers(self, project):
             return []
         return filter(bool, receivers.strip().splitlines())
 
-    def send_message(self, url, payload, receiver):
+    def get_receiver_threads(self, project):
+        receiver_threads = self.get_option('receiver_threads', project)
+        if not receiver_threads:
+            return []
+        return filter(bool, receiver_threads.strip().splitlines())
+
+    def send_message(self, url, payload, receiver, receiver_thread):
         payload['chat_id'] = receiver
         self.logger.debug('Sending message to %s ' % receiver)
+        if receiver_thread:
+          payload['message_thread_id'] = receiver_thread
+          self.logger.debug('Sending message to thread %s ' % receiver_thread)
```

Comment:
Please, put this fragment in order with the project's code style, in part of indents. The remaining codebase is with 4-spaces.

```suggestion
            payload['message_thread_id'] = receiver_thread
            self.logger.debug('Sending message to thread %s' % receiver_thread)
```